Description: Fix short description in manpages
 Fix short description in manpages
 .
Author: Pierre Chifflier <pollux@debian.org>

---
--- libcap-ng-0.6.6.orig/docs/capng_set_caps_fd.3
+++ libcap-ng-0.6.6/docs/capng_set_caps_fd.3
@@ -1,6 +1,6 @@
 .TH "CAPNG_SET_CAPS_FD" "3" "June 2009" "Red Hat" "Libcap-ng API"
 .SH NAME
-capng_set_caps_fd \- 
+capng_set_caps_fd \- write file-based capabilities to extended attributes
 .SH "SYNOPSIS"
 .B #include <cap-ng.h>
 .sp
--- libcap-ng-0.6.6.orig/docs/capng_get_caps_fd.3
+++ libcap-ng-0.6.6/docs/capng_get_caps_fd.3
@@ -1,6 +1,6 @@
 .TH "CAPNG_GET_CAPS_FD" "3" "June 2009" "Red Hat" "Libcap-ng API"
 .SH NAME
-capng_get_caps_fd \- 
+capng_get_caps_fd \- read file-based capabilities from extended attributes
 .SH "SYNOPSIS"
 .B #include <cap-ng.h>
 .sp
